include/materials/ADComputeISoilStress.h

```diff
@@ -155,6 +155,24 @@ class ADComputeISoilStress : public ADComputeFiniteStrainElasticStress<compute_s
   /// for effeciency.
   unsigned int _current_id;
 
+  /// The mean pressure of an element.
+  ADReal mean_pressure;
```

Comment:
As per the moose/mastodon naming convention, all variables defined in the .h file should start with an underscore. Sorry for not pointing this out earlier.